src/points/PointsHome.tsx

```diff
@@ -1,23 +1,23 @@
 import { NativeStackScreenProps } from '@react-navigation/native-stack'
```

Comment:
Are there meant to be no functional changes in this file?

Reply:
yes i think i moved the changes to a different PR but had a hard time turning off the vs code organize imports setting. i've removed these unrelated changes now